atomic_reactor/schemas/container.json

```diff
@@ -109,6 +109,7 @@
               "additionalProperties": false,
               "required": ["repo", "ref"]
             },
+            "operator_manifest": {"$ref": "#/definitions/operator_manifest"},
```

Comment:
Why does `container.json` have 2 anyOf schemas? Is this the right place for `operator_manifests`?

Reply:
Because either 'go' or 'remote_source' may be used, but not both.

Reply:
I see, thanks